### ISSUE TITLE
Feature request: `case` without curly braces

### ISSUE THREAD

**Issue by gharlan** [FriendsOfPHP/PHP-CS-Fixer#1540]
To avoid something like this:

``` php
switch ($foo) {
    case 'bar': {
        // ...
    }
}
```

**Comment by SpacePossum** [comment]
:+1: 
(I would like have a fixer to do the reverse, so I could base it on the one you want ;)  )

**Comment by keradus** [comment]
wow, any framework uses braces for case?
just curious

**Comment by SpacePossum** [comment]
don't know of any major framework, I just keep using those ^-^

**Comment by keradus** [comment]
ok, I'm good with it ;)
btw, you need to break a case before closing `}` ?

**Comment by SpacePossum** [comment]
>btw, you need to break a case before closing } ?

not needed, this;
```php
<?php
$a = 0;
switch($a) {
    case 0 : {
        echo '0';
    }
    break;
    default : {
        echo 'test';
    }
}
```
acts the same as:
```php
<?php
$a = 0;
switch($a) {
    case 0 : {
        echo '0';
        break;
    }
    default : {
        echo 'test';
    }
}
```

**Comment by keradus** [comment]
so you still need the break, if you could ommit it (if closing `}` acts as break) it would be cool ;)

**Comment by julienfalque** [comment]
Actually, these extra braces can appear anywhere, not only around `case` body ([example](https://3v4l.org/G2gZK)). This is why having the `break` statement inside or outside the braces has the same result.

AFAIK, these braces have no special meaning (no variable scope or something similar) and I think the fixer should be aware of that and remove them all.

**Comment by keradus** [comment]
just be aware of `$array{$index}` :D

**Comment by keradus** [comment]
Closed by #2929
